src/Telegram.Bot/Requests/Available methods/Messages/SetMessageReactionRequest.cs

```diff
@@ -0,0 +1,57 @@
+// ReSharper disable once CheckNamespace
+using Telegram.Bot.Requests.Abstractions;
+
+namespace Telegram.Bot.Requests;
+
+/// <summary>
+/// Use this method to change the chosen reactions on a message. Service messages can't be reacted to.
+/// Automatically forwarded messages from a channel to its discussion group have the same
+/// available reactions as messages in the channel.
+/// Returns <see langword="true"/> on success.
+/// </summary>
+[JsonObject(MemberSerialization.OptIn, NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+public class SetMessageReactionRequest : RequestBase<bool>,
+    IChatTargetable
+{
+    /// <inheritdoc />
+    [JsonProperty(Required = Required.Always)]
+    public ChatId ChatId { get; }
+
+    /// <summary>
+    /// Identifier of the target message. If the message belongs to a media group, the reaction
+    /// is set to the first non-deleted message in the group instead.
+    /// </summary>
+    [JsonProperty(Required = Required.Always)]
+    public int MessageId { get; }
+
+    /// <summary>
+    /// New list of reaction types to set on the message. Currently, as non-premium users, bots can
+    /// set up to one reaction per message. A custom emoji reaction can be used if it is either
+    /// already present on the message or explicitly allowed by chat administrators.
+    /// </summary>
+    [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+    public ReactionType[]? Reaction { get; set; }
```

Comment:
Change it to `IEnumerable<ReactionType>`, please. We use it everywhere in the inputs